content-src/asrouter/templates/SimpleSnippet/SimpleSnippet.schema.json

```diff
@@ -50,6 +50,10 @@
         "url": {
           "type": "string",
           "description": "The url where the link points to."
+        },
```

Comment:
Can you update the version of the template as well since this adds new functionality?